locales/de/subtitles/anita-malavasi/anita-malavasi-01.yml

```diff
@@ -9,56 +9,49 @@ de:
           s: 1
           from: 00:00:00,18
           to: 00:00:07,15
-          text: I was born in Roncolo di Quattro Castella on the 21st of May 1921.
+          text: Ich wurde am 21. Mai 1921 in Roncolo di Quattro Castella geboren.
         sequence-2:
           s: 2
           from: 00:00:07,15
           to: 00:00:14,27
-          text: We were a family of farmers, but our roots were different.
+          text: Wir waren eine Bauernfamilie, aber wir hatten andere Wurzeln.
         sequence-3:
           s: 3
           from: 00:00:14,27
           to: 00:00:22,23
-          text: My grandfather had studied to become a music teacher and had founded
-            the band of Vezzano.
+          text: Mein Großvater hatte studiert, um Musiklehrer zu werden und hatte die Musikgruppe von Vezzano gegründet.
         sequence-4:
           s: 4
           from: 00:00:22,23
           to: 00:00:27,01
-          text: My grandmother was the daughter of the town-clerk of Vezzano.
+          text: Meine Großmutter war die Tochter eines Stadtangestellten von Vezzano.
```

Comment:
How about the `Stadtschreiber`? This seems to be the 1-1 translation both from the English `town-clerk` and the Italian `segretario comunale`. I wasn't really sure, what a `Stadtschreiber` is, so I've looked it up [ on wikipedia]( https://de.wikipedia.org/wiki/Stadtschreiber_(Kanzleivorsteher))...